src/pages/Account.vue

```diff
@@ -61,16 +63,54 @@
 
 <script>
     import Store from '../store.js';
+    import NetMgr from '../services/netMgr.js';
     export default {
         name: 'account',
         data() {
             return {
-                voucherPayments: Store.trader.pendedVouchers
+                voucherPayments: Store.trader.pendedVouchers,
+                errorMessage : Store.error,
+                goodFeedback : false
             }
         },
         created: function () {
 
         },
+        methods: {
+            onRequestSubmissionEmail : function(event) {
+                var url = '/traders/' + Store.trader.id + '/voucher-history-email';
+                this.requestEmailBeSent(url,{"submission_date" : event.target.id})
+            },
+            onRequestVoucherHistoryEmail: function() {
```

Comment:
Do we necessarily want these to be different? i.e. not just the same but with null date.